docs/release_process.md

```diff
@@ -117,13 +104,23 @@ Before deploying to production, we need to assess the current state of our work
 
 ### Update Production Environment Variables
 
-In the cases where we need to update or add new environment variables, we need to get help from SRE:
+Environment variables are configured in 2 places:
+
+- Secret env vars via [Secret Manager](https://console.cloud.google.com/security/secret-manager)
+- Non-secret env vars via Helm [ConfigMap](https://github.com/mozilla/webservices-infra/blob/e8301f4b257550787098a25b078da99765e7a81a/monitor/k8s/monitor-www)
```

Comment:
In principle I think it's good to use permalinks usually, but in this case I think there might be a risk of someone trying to find the current value of a variable, and ending up with an old one.

And if the path does change in the future, I think we'd rather know right away by running into a dead link, than being under the impression that this documentation is still accurate.

```suggestion
- Non-secret env vars via Helm [ConfigMap](https://github.com/mozilla/webservices-infra/tree/main/monitor/k8s/monitor-www)
```